.github/workflows/compiler-warnings.yml

```diff
@@ -57,7 +59,11 @@ jobs:
 
     - name: Install dependencies
       run: |
-        .github/workflows/scripts/install-build-deps.sh
+        sudo apt update && \
```

Comment:
By not using the script, you are missing fail-retry mechanism for packages installation. It's there for a reason - sometimes Github share workers connectivity degrades and jobs start failing really frequently without fail-retry. I recommend keeping that.

Reply:
Not applicable anymore